tests/integration/monitors/run.rs

```diff
@@ -6,13 +6,27 @@ fn command_monitors_run() {
         EndpointOptions::new("POST", "/api/0/monitors/foo-monitor/checkins/", 200)
             .with_response_file("monitors/post-monitors.json"),
     );
+
     if cfg!(windows) {
         register_test("monitors/monitors-run-win.trycmd");
     } else {
         register_test("monitors/monitors-run.trycmd");
     }
 }
 
+#[test]
+fn command_monitors_run_token_auth() {
+    let _server = mock_endpoint(
+        EndpointOptions::new("POST", "/api/0/monitors/foo-monitor/checkins/", 200)
+            .with_response_file("monitors/post-monitors.json"),
+    );
+    if cfg!(windows) {
+        register_test("monitors/monitors-run-token-auth-win.trycmd").env("SENTRY_DSN", "");
```

Comment:
Ahh, I was trying to clear it out like this but that wasn't working since even having a blank value was reading it :-)

There must be some logic somewhere that treats empty as None in the new code

Reply:
Yup, it's here - https://github.com/getsentry/sentry-cli/blob/ac6532fc1aa69ff9f9d2bf74f4a6fe0b09f67c89/src/config.rs#L407-L416
Empty string fails to `.parse()` as `Dsn` type.